src/components/App.js

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
-/* eslint-disable no-unused-vars */
 import calculate from '../logic/calculate';
 
-function App() {
-  return (
-    <>
-      <Display result="" />
-      <ButtonPanel />
-    </>
-  );
+class App extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      total: '',
+      next: '',
+      operation: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(buttonName) {
+    console.log(buttonName);
```

Comment:
- [ ] It will be great to remove any `console.log` in your code. This ensures that any sensitive data in your application is not exposed